models/user.rb

```diff
@@ -61,12 +61,17 @@ def to_hash(params={})
           ).count
 
           # Note that the comments may have been responses to a thread not started by author_id.
-          comment_thread_ids = Comment.where(
+
+          # We need to include the comment_thread in thie query to prevent an N+1 query
+          # when we iterate through all of the comments to get their thread context
+          comment_thread_ids = Comment.includes(:comment_thread).where(
```

Comment:
I'm slightly confused by this query. I may be missing something basic, but based on the docs for [eager loading](http://docs.mongodb.org/ecosystem/tutorial/ruby-mongoid-tutorial/#eager-loading) I would expect to see us do something with each comment's `comment_thread` sub-document. Where are we using the comment_thread?